packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -248,19 +248,6 @@ class ServiceConnectionManager {
 
     service.onEvent(serviceStreamName).listen(handleServiceEvent);
 
-    _connectedState.value = const ConnectedState(true);
```

Comment:
why was all this code moved to be after the streams?

Reply:
never mind, this seems fine. If all this was being called before the stream listeners were initialized, then it shouldn't matter to be called after the stream listeners are initialized unawaited.